pages/Configuring/Keywords.md

```diff
@@ -102,7 +102,7 @@ The `name` can be easily obtained by doing `hyprctl devices`.
 Inside of it, put your config options. All options from the `input` category
 (and all subcategories, e.g. `input:touchpad`) can be put inside, **EXCEPT**:
 
-force_no_accel, follow_mouse, float_switch_override_focus, scroll_factor
+force_no_accel, follow_mouse, float_switch_override_focus
```

Comment:
you didn't add a per-device variable for this, this change is wrong

Reply:
apologies @vaxerski I'm not sure what your ask is.  should I back out this particular change?

Reply:
I see what you're saying, this isn't a per-device input config.  I backed out the change.